confluent_kafka/admin/__init__.py

```diff
@@ -35,6 +38,15 @@ class ConfigSource(Enum):
     DEFAULT_CONFIG = CONFIG_SOURCE_DEFAULT_CONFIG  #:
 
 
+class PurgeStrategy(Enum):
```

Comment:
While this technically maps nicely to librdkafka, I think from a user-experience it might be better to simply go with optional boolean kwargs: `purge(in_queue=True, in_flight=True, blocking=True)` (those are the defaults)